modules/nupac.nu

```diff
@@ -34,6 +34,7 @@ def get-flag-value [
 # Path where packages will be installed, can be changed to any other directory provided it is added to NU_LIB_DIRS variable
 def scripts-path [] {
     $env
+    |get -i "NUPAC_DEFAULT_LIB_DIR"
     |get -i "NUPAC_DEFAULT_LIB_DIR" 
```

Comment:
please remove one of them, something went wrong with the merge ig

Reply:
Done